pkg/csi_driver/node.go

```diff
@@ -157,6 +196,17 @@ func validateVolumeAttributes(attr map[string]string) error {
 	return nil
 }
 
+func validateSmbNodePublishSecrets(secrets map[string]string) error {
+	if secrets[smbUser] == "" {
```

Comment:
does smb have any restrictions on what are valid values?

Reply:
There's some restrictions but they're a bit convoluted and are caught during mount so I think checking they're not empty is good enough.
Example: the SMB username shouldn't be more than 20 characters long, not counting the domain name. Special characters, etc.

Passwords have no restriction.